oslc_prolog/lib/oslc_core.pl

```diff
@@ -48,12 +47,46 @@
         atom_concat('oslc.', OP, Key),
         Option =.. [OP, Value]
       ), Options
-    )
-  ; Options = []
+    ),
+    L1 = [],
+    ( member(rdfs=sup, Search)
+    -> findall(SuperClass, (
+         rdfs_subclass_of(IRI, SuperClass),
+         IRI \= SuperClass
+       ), SuperClasses),
+       append(L1, SuperClasses, L2)
+    ; L2 = L1
+    ),
+    ( member(rdfs=sub, Search)
+    -> findall(SubClass, (
+         rdfs_subclass_of(SubClass, IRI),
+         IRI \= SubClass
+       ), SubClasses),
+       append(L2, SubClasses, L3)
+    ; L3 = L2
+    ),
+    ( member(rdfs=ind, Search)
+    -> findall(Individual, (
+         rdfs_individual_of(Individual, IRI),
+         IRI \= Individual
+       ), Individuals),
+       append(L3, Individuals, L4)
+    ; L4 = L3
+    ),
+    ( member(rdfs=cls, Search)
+    -> findall(Class, (
+         rdfs_individual_of(IRI, Class),
+         IRI \= Class
+       ), Classes),
+       append(L4, Classes, L5)
+    ; L5 = L4
+    ),
+    list_to_set(L5, Set)
+  ; Options = [],
+    Set = []
```

Comment:
FEATURE: query any resource though GET with the following parameters:
* `<resource>?rdfs=sup` - to get all super classes of the <resource> also
* `<resource>?rdfs=sub` - to get all subclasses of the <resource> also
* `<resource>?rdfs=ind` - to get all individuals of the <resource> also
* `<resource>?rdfs=cls` - to get all classes the <resource> is inheriting from also